### PR TITLE
Web Dashboard WG: Document mailing list

### PR DESCRIPTION
**Pull request by padovan**
Add info about the mailing list we just created for the Web Dashboard Working Group.

### CODE REVIEW

**Comment by gctucker** [comment]
Also please use a fork for making PRs rather than pushing branches directly to the main repository.

File: kernelci.org/content/en/docs/org/working-groups.md [hunk 1]
```diff
@@ -40,6 +40,8 @@ of what is required.  Based on this, we'll then be looking for technical
 solutions including existing frameworks, dashboards used by other projects and
 potentially a new design from scratch using modern web technology.
 
+The Web Dashboard WG has a [mailing list](https://kernelci.org/docs/org/working-groups/#web-dashboard) at the [address](mailto:<kernelci-webdashboard@groups.io>).
```
Comment (patersonc) [inline kernelci.org/content/en/docs/org/working-groups.md:43]: I'm not sure why "mailing list" links to where it does, given that the user is already reading the section that the link goes to.

I would assume that a link for "mailing list" would link to the actual mailing list?

Comment (gctucker) [reply kernelci.org/content/en/docs/org/working-groups.md:43]: Also, an absolute URL is not a good idea in general.  A relative path would be better - but yes I agree with Chris and this isn't useful here in any case.

**Review by gctucker — COMMENTED** [review]
How about this instead?
```
## Web dashboard

**Workboard:** https://github.com/orgs/kernelci/projects/4
**Mailing list:** [kernelci-webdashboard@groups.io](mailto:<kernelci-webdashboard@groups.io>)
```
which would render as:

## Web dashboard

**Workboard:** https://github.com/orgs/kernelci/projects/4
**Mailing list:** [kernelci-webdashboard@groups.io](mailto:<kernelci-webdashboard@groups.io>)

File: kernelci.org/content/en/docs/org/working-groups.md [hunk 2]
```diff
@@ -19,6 +19,7 @@ as:
 ## Web dashboard
 
 **Workboard:** https://github.com/orgs/kernelci/projects/4
+**Mailing list:** [kernelci-webdashboard@groups.io](mailto:<kernelci-webdashboard@groups.io>)
```
Comment (gctucker) [inline kernelci.org/content/en/docs/org/working-groups.md:22]: It's on staging:
https://static.staging.kernelci.org/docs/org/working-groups/#web-dashboard

However, it appears on the same line as **Workboard:**.  So maybe a line break, or a list or a paragraph is needed here.  I think the ideal syntax would be a definition list:
https://www.markdownguide.org/extended-syntax/#definition-lists

**Review by gctucker — APPROVED** [review]

**Comment by gctucker** [comment]
@nuclearcat Maybe you could add the sysadmin working group email too?

**Pull request merged**